### PR TITLE
Updated docs for MySQL and PostgreSQL connectors to include SHOW COLU…

### PR DESCRIPTION
**Pull request by brian-rickman**
Updated docs for MySQL and PostgreSQL connectors to include SHOW COLUMNS examples, and known limitations.

### CODE REVIEW

File: presto-docs/src/main/sphinx/connector/mysql.rst [hunk 1]
```diff
@@ -44,9 +44,23 @@ in this database by running ``SHOW TABLES``::
...
 If you used a different name for your catalog properties file, use
 that catalog name instead of ``mysql`` in the above examples.
+
+MySQL Connector Limitations
+---------------------------
+
+The following SQL statements are not supported in this version of the connector::
+
+    CREATE TABLE
+    DROP TABLE
```
Comment (cawallin) [inline presto-docs/src/main/sphinx/connector/mysql.rst]: I believe there is a config variable that controls DROP TABLE: allow-drop-table
If it is set to true in your mysql.properties or postgresql.properties file, you should be able to drop tables.

File: presto-docs/src/main/sphinx/connector/mysql.rst [hunk 2]
```diff
@@ -44,9 +44,23 @@ in this database by running ``SHOW TABLES``::
...
 
 If you used a different name for your catalog properties file, use
 that catalog name instead of ``mysql`` in the above examples.
+
+MySQL Connector Limitations
+---------------------------
+
+The following SQL statements are not supported in this version of the connector::
+
+    CREATE TABLE
```
Comment (cawallin) [inline presto-docs/src/main/sphinx/connector/mysql.rst]: I haven't tried this myself, but in looking at the code, CREATE TABLE should work. What error/stacktrace does this give?

**Comment by brian-rickman** [comment]
@cawallin: presto:default> create table mysql.test.test_table (c1 bigint);
Query 20150810_190230_00010_nxrzs failed: This connector does not support creating tables

**Comment by electrum** [comment]
Merged, thanks!

**Pull request closed without merging**